apps/web/cypress/tests/auth.spec.ts

```diff
@@ -38,6 +40,63 @@ describe('User Sign-up and Login', function () {
       cy.getByTestId('submitButton').click();
       cy.get('.mantine-TextInput-error').contains('Please provide a valid email');
     });
+
+    it('should allow to sign-up with Github, logout, and login', function () {
+      const isCI = Cypress.env('IS_CI');
+      if (!isCI) return;
```

Comment:
All the e2e tests that use GH button will be only running on the CI, because we don't want all the contributors to have to setup the GH app and credentials etc.